internal/providers/pluginfw/pluginfw.go

```diff
@@ -62,6 +63,7 @@ func (p *DatabricksProviderPluginFramework) DataSources(ctx context.Context) []f
 		sharing.DataSourceShare,
 		sharing.DataSourceShares,
 		catalog.DataSourceFunctions,
+		user.DataSourceUsers,
```

Comment:
You'll need to merge `main` to merge this PR. Make sure you add this in the list of not-migrated datasources.

Reply:
Done 👍